account_wallet_sale/models/sale_order_line.py

```diff
@@ -0,0 +1,38 @@
+# Copyright 2020 ACSONE SA/NV (http://www.acsone.eu)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+from odoo import api, fields, models
+
+
+class SaleOrderLine(models.Model):
+    _inherit = 'sale.order.line'
+
+    account_wallet_id = fields.Many2one(
+        comodel_name='account.wallet',
+        ondelete='restrict',
+        index=True,
+    )
+
+    @api.model
+    def create(self, vals):
```

Comment:
```suggestion
    @api.model_create_multi
    def create(self, vals_list):
```

Reply:
Not that one because I won't be able to filter recordset on which reapply() after call to super()